compiler/tests/fail/stack_allocation/x86-64/range_overflow.jazz

```diff
@@ -1,9 +1,7 @@
-export
-fn f () -> reg u64 {
-  stack u64[2] a, b;
-  reg u64 res;
-  a[0] = 1;
-  b[1:1] = a[0:1];
-  res = b[1];
-  return res;
+export fn main (reg u64 x) -> reg u64 {
```

Comment:
The error message it not clear (range/slice) this is internal code.

Reply:
proposed message "Cannot access to the subarray [16;32[ of  a.201[0; 24[, the access overflow, your program is probably unsafe"

Reply:
I followed your suggestion.
```
"tests/fail/stack_allocation/x86-64/range_overflow.jazz", line 4 (2-13):
compilation error in function main:
stack allocation: cannot access the subarray [16; 32[ of a.200[0; 24[, the access overflows, your program is probably unsafe
```